StandardAnalysis/python/HistogramDefinitions.py

```diff
@@ -1180,7 +1180,7 @@
     inputCollection = cms.vstring("muons", "muons"),
     histograms = cms.VPSet (
         cms.PSet (
-            name = cms.string("diMuonPt"),
+            name = cms.string("diMuonPtWide"),
```

Comment:
will this always be "wide" from now on?

Reply:
It actually should have been something+wide before. Whenever you used the histSetsMuon option, it would complain about a duplicate histogram in the Muon-muon folder, and that is the culprit. I just changed the name for 1. that warning to go away and 2. to have a wider di muon pt to match the run 2 AN plots.